docs/content/manual/rancher-integration/drain-using-rancher-ui.md

```diff
@@ -0,0 +1,232 @@
+
+
+### 6. Longhorn volume with 50/100 Gi data
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND 1 volume with 50/100 Gi of data attached to a node
+
+**When** Drain the node using Rancher UI with default value and `Delete Empty Dir Data` enabled
+
+**Then** Drain should be successful for 2 node but should fail for last node
+
+AND Volume data should be intact
+
+AND Once there are 2 healthy replica rebuilt, then retry drain should be successful
+
+
+### 7. Drain the attached node when backup in Longhorn is in progress
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND 1 RWO volume (3 replicas) attached with 10 Gi data
+
+AND Backup is in progress
+
+**When** Drain the attached node using Rancher UI with default value and `Delete Empty Dir Data` enabled
+
+**Then** Drain should be successful
+
+AND Backup should be interrupted
+
+AND Volume's data should be intact
+
+AND Volume should be attached once the drain finishes
+
+AND Next backup should be successful
+
+### 8. Drain the replica node when backup in Longhorn is in progress
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND 1 RWO volume (3 replicas) attached with 10 Gi data
+
+AND Backup is in progress
+
+**When** Drain the node on which replica responsible for backup exists using Rancher UI
+
+**Then** Drain should be successful
+
+AND Backup should be interrupted
+
+AND Volume's data should be intact
+
+AND Volume should be attached once the drain finishes
+
+AND Next backup should be successful
+
+
+### 9. Drain operation with short timeout
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exist
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with timeout 5 sec
+
+**Then** Drain should be interrupted due to timeout
+
+AND There shouldn't be any data loss in the volumes
+
+AND No volumes should stuck in attaching/detaching state
+
+
+### 10. Interrupt drain operation
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exist
+
+AND few RWO and RWX volumes unattached
+
+**When** Trigger the drain and cancel it when drain is in progress 
+
+**Then** Drain should be interrupted without causing data loss of volumes.
+
+AND No volumes should stuck in attaching/detaching state
+
+
+Test with Longhorn setting of 'Node Drain Policy': `allow-if-replica-is-stopped`
+--------
+
+### 1. Drain using Rancher UI - Single replica:
+
+**Given** Single node cluster with Longhorn installed
+
+AND 1 RWO and 1 RWX volumes unattached
+
+
+**When** Drain the node using Rancher UI default value and `Delete Empty Dir Data` enabled
+
+**Then** Drain should be successful
+
+AND Volumes should be unattached and data should be intact after drain completes
```

Comment:
Then, after the node drain, I think we should uncordon the node using the Rancher UI.  Afterwards, we could verify that the volumes data is intact by checking that the data is still accessible. Therefore, we should modify the test step as follows:

Then Uncordon the worker node using the Rancher UI
AND volumes data is intact after uncordon completes